src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessStartInfoTests.cs

```diff
@@ -186,5 +188,59 @@ public void TestUseShellExecuteProperty()
             // Calling the getter
             Assert.False(psi.UseShellExecute, "UseShellExecute=true is not supported on onecore.");
         }
+
+        [Fact]
+        public void TestArgumentsProperty()
+        {
+            ProcessStartInfo psi = new ProcessStartInfo();
+            Assert.Equal(string.Empty, psi.Arguments);
+
+            psi = new ProcessStartInfo("filename", "-arg1 -arg2");
+            Assert.Equal("-arg1 -arg2", psi.Arguments);
+        }
+
+        [Fact]
+        public void TestCreateNoWindowProperty()
+        {
+            Process testProcess = CreateProcessInfinite();
+            try
+            {
+                testProcess.StartInfo.CreateNoWindow = true;
```

Comment:
Should this be a [Theory] that tests both true and false values?  I assume the purpose of this test is just to ensure that we can actually start the process successfully in either case, rather than validating whether another window did or did not appear?